package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "dependencies": {
     "@databraid/github-widget": "^1.0.16",
-    "@databraid/slack-widget": "^1.0.14",
-    "@databraid/transit-widget": "^1.0.21",
+    "@databraid/slack-widget": "^1.0.17",
```

Comment:
these shouldn't have ^. 

Also are these the latest versions of these npm modules?

Reply:
Removed ^. As of the submission of this pull request on Friday these were the latest versions.